packages/manager/modules/pci/src/projects/project/storages/databases/components/plans-list/translations/Messages_fr_FR.json

```
@@ -10,5 +10,7 @@
   "pci_database_plans_list_spec_ram_unique": "{{ram}} RAM",
   "pci_database_plans_list_spec_core_unique": "{{core}} VCores",
   "pci_database_plans_list_spec_storage_unique": "{{storage}} de stockage SSD",
-  "pci_database_plans_list_current_offer": "Offre actuelle"
+  "pci_database_plans_list_current_offer": "Offre actuelle",
+  "pci_database_plans_list_spec_private_network": "Réseaux privés",
+  "pci_database_plans_list_spec_price_unit": " / heure / nœud"
```

may be it could be better if we have a dynamic value here:
```suggestion
  "pci_database_plans_list_spec_price_unit": "{{ price }} / heure / nœud"
```

---

We use the `ovh-manager-catalog-price` component to display price, so I don't think we can include it in the translation.